website/addons/dropbox/tests/factories.py

```diff
@@ -25,3 +22,9 @@ class DropboxNodeSettingsFactory(ModularOdmFactory):
     owner = SubFactory(ProjectFactory)
     user_settings = SubFactory(DropboxUserSettingsFactory)
     folder = 'Camera Uploads'
+
+class DropboxAccountFactory(ExternalAccountFactory):
+    provider = 'dropbox'
+    provider_id = Sequence(lambda n: 'id-{0}'.format(n))
+    oauth_key = Sequence(lambda n: 'key-{0}'.format(n))
+    oauth_secret = Sequence(lambda n: 'secret-{0}'.format(n))
```

Comment:
The `oauth_secret` was removed from `BoxAccountFactory`. Should it also be removed here?